build/Targets/ProducesNoOutput.Imports.targets

```diff
@@ -7,8 +7,9 @@
     <TargetPath></TargetPath>    <!-- Prevent projects referencing this from trying to pass us to the compiler -->
   </PropertyGroup>
   
-  <Target Name="CoreCompile" />                               <!-- Prevent Csc from being called -->  
-  <Target Name="GenerateTargetFrameworkMonikerAttribute" />   <!-- Don't generate TFM attribute -->  
+  <!-- Return this back when up-to-date bug is fixed (projects are being build everytime) -->
```

Comment:
Bill's already checked this in. If you sync, you won't need this change.

---
In reply to: [74526841](https://github.com/dotnet/roslyn-project-system/pull/349#discussion_r74526841) [](ancestors = 74526841,74463316)